src/Factories/Api/SurveyData.php

```diff
@@ -52,12 +72,18 @@ public function post(array $fields, string $return_format, bool $raw = false)
             $data['cond'] = $this->condition;
         }
 
+        foreach (['start', 'end'] as $prop) {
```

Comment:
I'm not objecting to this and I assume you've done it for future-proofing, but it does feel a little odd when it's only two properties being iterated over.

Reply:
Potential future proofing (although not sure how many more props the Decipher API exposes which are useful), but also just because methods were getting long with repeated if() blocks